src/renderer/components/swap/Swap.tsx

```diff
@@ -88,7 +88,7 @@ export type SwapProps = {
   targetWalletAddress: O.Option<Address>
   onChangePath: (path: string) => void
   network: Network
-  sourcePoolRouter: O.Option<string>
+  sourcePoolRouter: string
```

Comment:
Pls revert `sourcePoolRouter` back to `O.Option<string>`, so we do know if it's still loading. Because `'''` is used whenever source asset is RUNE (no pool address needed in this case). Also to reflect loading state, it's `O.none` in this case.